src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderResource.java

```diff
@@ -242,6 +243,21 @@ public InmateDetail transferInPrisoner(
         return prisonerReleaseAndTransferService.transferInPrisoner(offenderNo, requestToTransferIn);
     }
 
+    @ApiResponses({
+        @ApiResponse(code = 400, message = "Invalid request.", response = ErrorResponse.class),
+        @ApiResponse(code = 403, message = "Forbidden - user not authorised to transfer a prisoner", response = ErrorResponse.class),
+        @ApiResponse(code = 404, message = "Requested resource not found.", response = ErrorResponse.class),
+        @ApiResponse(code = 500, message = "Unrecoverable error occurred whilst processing request.", response = ErrorResponse.class)})
+    @ApiOperation("Transfer a prisoner into a prison from court. Must be an out prisoner in currently in transfer status, requires the TRANSFER_PRISONER role")
```

Comment:
does this also require a token with a username? if so we could add this to the doc. 

The existing calls don't mention that this is required but should, they also fail in strange ways when not present. 
Will this fail in strange ways if not provided? - do we have a test for that case?

Reply:
@andrewrlee this endpoint is based on "/{offenderNo}/transfer-in".The test is here OffendersResourceCourtTransfersTest but nothing added for 403 scenario